### PR TITLE
update balance

### PR DESCRIPTION
**Pull request by bowenwang1996**
Update balance after execution a function call in runtime.

### CODE REVIEW

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -290,30 +289,37 @@ impl Runtime {
...
+            );
+            let wasm_res = executor::execute(
+                &sender.code,
+                &method_name,
+                args,
+                &[],
+                &mut runtime_ext,
+                &wasm::types::Config::default(),
+                &RuntimeContext::new(
+                    sender.amount,
```
[the remark this answers is not in the document]
Comment (bowenwang1996) [reply node/runtime/src/lib.rs]: Fixed in 0978846

File: node/runtime/src/lib.rs [hunk 2]
```diff
@@ -518,36 +524,47 @@ impl Runtime {
...
+            );
+            let wasm_res = executor::execute(
+                &receiver.code,
+                &async_call.method_name,
+                &async_call.args,
+                &[],
+                &mut runtime_ext,
+                &wasm::types::Config::default(),
+                &RuntimeContext::new(
+                    receiver.amount,
```
[the remark this answers is not in the document]
Comment (bowenwang1996) [reply node/runtime/src/lib.rs]: Fixed in 0978846

File: node/runtime/src/lib.rs [hunk 3]
```diff
@@ -554,7 +563,7 @@ impl Runtime {
                 sender_id,
                 receiver_id,
             );
-            if let Ok(_) = result {
+            if result.is_ok() {
                 receiver.amount = wasm_res.balance;
```
Comment (evgenykuzyakov) [inline node/runtime/src/lib.rs]: This one is the amount without stake. You should add stake back

Comment (bowenwang1996) [reply node/runtime/src/lib.rs]: Fixed in 0354228

**Pull request merged**